example/transform/self_instruct_pdf_source.ipynb

```diff
@@ -114,19 +125,104 @@
nd I wrote a signup program that ensures all the appointments within a given set of office hours are clustered at the end. Because they come at the end of my day these meetings are never an interruption. (Unless their working day ends at the same time as mine, the meeting presumably interrupts theirs, but since they made the appointment it must be worth it to them.) During busy periods, office hours sometimes get long enough that they compress the day, but they never interrupt it. When we were working on our own startup, back in the 90s, I evolved another trick for partitioning the day. I used to program from dinner till about 3 am every day, because at night no one could interrupt me. Then I\\'d sleep till about 11 am, and come in and work until dinner on what I called \"business stuff.\" I never thought of it in these terms, but in effect I had two workdays each day, one on the manager\\'s schedule and one on the maker\\'s. When you\\'re operating on the manager\\'s schedule you can do something you\\'d never want to do on the maker\\'s: you can have speculative meetings. You can meet someone just to get to know one another. If you have an empty slot in your schedule, why not? Maybe it will turn out you can help one another in some way. Business people in Silicon Valley (and the whole world, for that matter) have speculative meetings all the time. They\\'re effectively free if you\\'re on the manager\\'s schedule. They\\'re so common that there\\'s distinctive language for proposing them: saying that you want to \"grab coffee,\" for example. Speculative meetings are terribly costly if you\\'re on the maker\\'s schedule, though. Which puts us in something of a bind. Everyone assumes that, like other investors, we run on the manager\\'s schedule. So they introduce us to someone they think we ought to meet, or send us an email proposing we grab coffee. At this point we have two options, neither of them good: we can meet with them, and lose half a day\\'s work; or we can try to avoid meeting them, and probably offend them. Till recently we weren\\'t clear in our own minds about the source of the problem. We just took it for granted that we had to either blow our schedules or offend people. But now that I\\'ve realized what\\'s going on, perhaps there\\'s a third option: to write something explaining the two types of schedule. Maybe eventually, if the conflict between the manager\\'s schedule and the maker\\'s schedule starts to be more widely understood, it will become less of a problem. Those of us on the maker\\'s schedule are willing to compromise. We know we have to have some number of meetings. All we ask from those on the manager\\'s schedule is that they understand the cost. **Thanks** to Sam Altman, Trevor Blackwell, Paul Buchheit, Jessica Livingston, and Robert Morris for reading drafts of this. **Related:** * How to Do What You Love\\n* Good and Bad Procrastination\\n* Turkish Translation\\n* French Translation\\n* Korean Translation']\n"
```

Comment:
same, shall we use pprint to print the output? And can we reduce the output length, it is too long here.